frontend-maven-plugin/src/main/java/com/github/eirslett/maven/plugins/frontend/mojo/InstallNodeAndPnpmMojo.java

```diff
@@ -74,32 +75,35 @@ protected boolean skipExecution() {
     @Override
     public void execute(FrontendPluginFactory factory) throws InstallationException {
         ProxyConfig proxyConfig = MojoUtils.getProxyConfig(session, decrypter);
-        String nodeDownloadRoot = getNodeDownloadRoot();
-        String npmDownloadRoot = getPnpmDownloadRoot();
+        // Use different names to avoid confusion with fields `nodeDownloadRoot` and
+        // `pnpmDownloadRoot`.
+        //
+        // TODO: Remove the `downloadRoot` config (with breaking change) to simplify download root
+        // resolution.
+        String resovledNodeDownloadRoot = getNodeDownloadRoot();
```

Comment:
**typo**: should be resolvedNodeDownloadRoot

Reply:
Fixed.